package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "@revenuecat/purchases-js",
-  "version": "0.9.0",
+  "version": "0.10.0",
```

Comment:
Did the release train miss this, and the same on line 9? (Not suggesting any changes for this PR.)

Reply:
The current automations are not updating the lock files here and for the sample apps. Need to fix those to also update these on each release... It shouldn't affect anything though I believe.